### ISSUE TITLE
Explain DATA and ANALYZABLE_DATA segments in results page

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#474]
The DATA and ANALYZABLE_DATA segment names in Sec. A2 and A3 of the results page are confusing. We should use the additional metadata in the segment table to explain what they are.

**Comment by duncan-brown** [comment]
@spxiwh says it should list four stages of data:

1. Science time.
2. Whatever is returned from the segment module (e.g. removes segments < 2000 seconds).
3. What is returned from the datafind module (includes data available on the cluster).
4. Data that actually gets analyzed by the inspiral program.

**Comment by spxiwh** [comment]
See here:

https://github.com/ligo-cbc/pycbc/blob/master/pycbc/workflow/analysislogging.py#L45

Note step 2 also includes removal of CAT_1 segments.

**Comment by ahnitz** [comment]
I think we should just show SCIENCE_OK (DATA) , and SCIENCE_OK - CAT1 (ANALYZABLE_DATA), and TRIGGERS_PRODUCED, which is what I think was intended, but not working. See the code here

https://github.com/ligo-cbc/pycbc/blob/master/pycbc/workflow/segment.py#L1336

I wrote this and I think I failed to remember to deepcopy the segmentlist so they ended up containing the same values. I would consider this a bug, and then the plots should contain the desired values.
